src/roktManager.ts

```diff
@@ -48,29 +92,28 @@ export default class RoktManager {
         }
 
         try {
-            return this.launcher.selectPlacements(options);
+            return this.kit.selectPlacements(options);
         } catch (error) {
             return Promise.reject(error instanceof Error ? error : new Error('Unknown error occurred'));
         }
     }
 
+    private isReady(): boolean {
+        return Boolean(this.kit);
+    }
```

Comment:
i suggest either setting a specific `isReady` key (inside of attachKit), or I'd be very explicit and put a comment here that says that the Rokt Manager is in a ready state only once a kit is attached.

it might be helpful to have a comment right above the class and link to the rokt web kit to discus the order of the flow, so that future developers can grok it more easily